### DISCUSSION TITLE
[Bug]: .tsx file attachments return "Error processing file"

### DISCUSSION THREAD

**Discussion by itsolidude** [danny-avila/LibreChat#4329] [Troubleshooting] [answered]
### What happened?

Uploading any file type other than .tsx works fine. But as soon as you try to upload a .tsx file it will throw you an error ""Error processing file". Same issue occurs on light and full. I cant imagine it being due to anything browser related but gave different browsers a shot anyways. The issue is most certainly caused by some file type conversion.

### Steps to Reproduce

1. Upload a .tsx file
2. An error popup will appear saying "Error processing file".

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Thanks, I confirmed this is an issue with the dependency we are using to load text-based files for embedding processing. Will need to implement a custom loader.

**Reply by cantalupo555** [reply]
Will we have the fix in the next update?

![image](https://github.com/user-attachments/assets/ba7bfbd9-b16e-49a7-a2f9-ee6746d0d86a)

**Reply by dx314** [reply]
Is this likely to be fixed soon? This is a dealbreaker for me. TSX files are just text files, I don't understand why they're being processed differently to any other application/* file?
